### ISSUE TITLE
canHandlePendingAuthorization is FALSE when Authorized should be done manually (Asychronous is activated)

### ISSUE THREAD

**Issue by ChristianAF** [saucal/woocommerce-gateway-amazon-pay#44]
![image](https://user-images.githubusercontent.com/43881685/105363044-60cd5680-5bda-11eb-8d8e-e29e01be1710.png)

Amazon settings:
![image](https://user-images.githubusercontent.com/43881685/105363081-6c208200-5bda-11eb-83ea-ccceddcb6b60.png)

**Comment by msaggiorato** [comment]
This seems to be an error in between the API docs and the API implementation.

You're correct to assume that this should work, but when set up like this (and according to the docs there's no restriction to do this, see screenshot below), it doesn't work.

![image](https://user-images.githubusercontent.com/6192180/105368651-852c3180-5be0-11eb-8f5f-6a37c885e889.png)

![image](https://user-images.githubusercontent.com/6192180/105368497-60d05500-5be0-11eb-92be-57d43d8c235c.png)

-----

Bottom line, there's no way of using these payment methods to test for async auths. We'd need to implement the new layer for testing, similar to async captures, which is out of scope imo.

I'd close this as wontfix for now.
